microsoft/botframework-solutions
# [TypeScript][Virtual Assistant Template] Add Skill folder

### Purpose
*What is the context of this pull request? Why is it being done?*
We added the `Skills` folder to maintain the consistency with C# template.

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp?
- Add the `Skills` folder in `Virtual Assistant Template`
- Update `Virtual Assistant Sample`
- Update `package-lock.json` file

### Tests
*Is this covered by existing tests or new ones? If no, why not?*
\-

### Testing Steps
1. Execute `yo botbuilder-assistant` and follow the prompts.
1. Verify that the `Skills` folder is copied to resources, that matches with the following image.

![image](https://user-images.githubusercontent.com/11904023/61087376-d401e400-a40b-11e9-944a-ca2e1591c250.png)

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*
\-
